diskqueue.go

```diff
@@ -346,30 +354,30 @@ func (d *diskQueue) readOne() ([]byte, error) {
 		d.reader = bufio.NewReader(d.readFile)
 	}
 
-	err = binary.Read(d.reader, binary.BigEndian, &msgSize)
+	err = binary.Read(d.reader, binary.BigEndian, &d.readMsgSize)
```

Comment:
nit: I would like to keep msgSize and use d.readMsgSize := msgSize, which might be easier for sync with upstream (https://github.com/nsqio/go-diskqueue) in the future.

Reply:
SG